tools/clang/test/CodeGenDXIL/hlsl/objects/NodeObjects/array-in-workgraphrecord-1.hlsl

```diff
@@ -0,0 +1,38 @@
+// RUN: %dxc -Tlib_6_8 %s | FileCheck %s
```

Comment:
We want this change in release, but I'm concerned that without a `// REQUIRES: dxil-1-8`, these tests will fail in downlevel validator testing.  When I manually tested for downlevel validator on main, they failed in this scenario.

Reply:
Ok, this will pass once this (https://github.com/microsoft/DirectXShaderCompiler/pull/6340/commits/30fb156d3f37fe6d2630d63f6a062549b2b78767) commit merges to release.